code/game/objects/objs.dm

```diff
@@ -53,6 +53,7 @@
 /obj/item/proc/is_used_on(obj/O, mob/user)
 
 /obj/proc/process()
+	set waitfor = 0

```

Comment:
This is so sleeps don't lock up the scheduler---though, I suppose I could just a `spawn` to `eat` for the singulo, but this seems like the better way to handle it.